meetings/Agenda_2017-12-11.md

```diff
@@ -0,0 +1,38 @@
+## Meeting Agenda, December 11, 2017
+
+1. **Welcome new (potential) exec**
+ * Nil and Ahmed are possibly joining exec!
+ * Brief discussion of roles / interests
+ * Anyone else know of interested people or good people to ask?
+
+2. **RQM debrief / next steps**
+ * Collect debrief notes from each person involved - where to put this?
+ * Plan for next year (Joel)
+ * Writing something up (Joel / Luke)
+ * Celebration party?
+
+3. **Coworking sessions**
+ * How are they going? Should we continue? Make changes to frequency / timing / location?
```

Comment:
- Learn by working on SO problems, or GitHub issues as a group instead?
- Mutual projects that span the semester as we discussed previously?